commit 23d3f7b96b3ffdb358a1b4d0e11800b7915a3074 (refs/changes/19/319/1)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2016-08-18T18:49:19-07:00 (3 years, 2 months ago)
    
    joyent/node-smartdc-auth#8 travis build broke (node 0.8 is dead, long live 0.8)

diff --git a/.travis.yml b/.travis.yml
index ea0cf7c..4a6432a 100644
--- a/.travis.yml
+++ b/.travis.yml
@@ -5,7 +5,5 @@ node_js:
   - "4.1"
   - "0.12"
   - "0.10"
-  - "0.8"
 before_install:
-  - "npm install -g npm@'>=1.4.3'"
   - "make check"
